src/polyfem/io/MshWriter.hpp

```diff
@@ -18,7 +18,10 @@ namespace polyfem::io
 		/// @param[in] binary output binary or not
 		static void write(
 			const std::string &path,
-			const mesh::Mesh &mesh,
+			const Eigen::MatrixXd &V,
```

Comment:
why not both?

Reply:
Sure, but we need to fix the Mesh3D data structure to not flip element order.